### PR TITLE
[10.0][MIG] geoengine_partner

### PR DESCRIPTION
**Pull request by ecino**
Migrate geoengine partner module

### CODE REVIEW

File: base_geoengine/geo_view/geo_raster_layer.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ class GeoRasterLayer(models.Model):
     url = fields.Char('Service URL', size=1024)
 
     # technical field to display or not wmts options
-    is_wmts = fields.Boolean(compute='_compute_is_wmts')
+    is_wmts = fields.Boolean(compute='_get_is_wmts')
```
Comment (yvaucher) [inline base_geoengine/geo_view/geo_raster_layer.py]: please keep `_compute` as it is an OCA convention for computed fields.

base_geoengine/geo_view/geo_raster_layer.py:38: [C8108(method-compute), GeoRasterLayer] Name of compute method should start with "_compute_"
base_geoengine/geo_view/geo_raster_layer.py:59: [C8108(method-compute), GeoRasterLayer] Name of compute method should start with "_compute_"

File: base_geoengine/geo_view/geo_raster_layer.py [hunk 2]
```diff
@@ -73,18 +73,17 @@ class GeoRasterLayer(models.Model):
         required=True)
     use_to_edit = fields.Boolean('Use to edit')
 
-    @api.multi
+    @api.one
```
Comment (yvaucher) [inline base_geoengine/geo_view/geo_raster_layer.py]: consider api.one as deprecated, there was some flip flop about it but the final result is that it is discouraged to use it.

**Review by yvaucher — CHANGES_REQUESTED** [review]
I started to review then figured our a lot of change were made in base_geoengine.

I guess you wanted to fix PEP8 issues. May I suggest you to do this in another PR?

**Comment by ecino** [comment]
Oops, you are right there are some unwanted changes here! I just tried to replace the `openerp` imports but I might have brought some unwanted changes I have to check what I did :smile:

**Comment by ecino** [comment]
@yvaucher I think it's better now!

**Comment by ecino** [comment]
It seems that odoo-sphinx-autodoc is creating errors in the travis test instance...

**Comment by ecino** [comment]
@yvaucher I guess you can review now. I just replaced openerp imports with odoo, otherwise it's only geoengine_partner changes.

**Review by yvaucher — APPROVED** [review]

**Pull request merged**